doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -274,6 +274,10 @@ Bluetooth samples
 
     * Fixed an interoperability issue with iOS devices by setting the report IDs of HID input and output reports to zero.
```

Comment:
Can we prefix commit message with `samples: peripheral_fast_pair:` ? (for consistency with existing commits that modify the sample)